src/dotnet-scaffolding/dotnet-scaffold-aspire/ScaffoldSteps/ValidateContextStep.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Scaffolding.Core.Scaffolders;
+using Microsoft.DotNet.Scaffolding.Core.Steps;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.Tools.Scaffold.Aspire.ScaffoldSteps;
+
+public class ValidateContextStep : ScaffoldStep
```

Comment:
What do you think about ValidateOptionsStep as a name instead? At the point this would execute, that's the only part of the context that would be relevant (the step is just itself, the property bag would be empty since no other steps have run). And really that's what you're using it for.

Reply:
yep thats a better name for sure.